src/pages/reference/guides.md

```diff
@@ -16,16 +18,20 @@ These are guides centered around deploying cool projects up on Railway.
 - [Deploy Website Analytics](https://blog.railway.app/p/self-hosted-website-analytics)
 
 ### Technical Guides
+
 - [ExpressJS with Postgres](https://blog.railway.app/p/expressjs-with-postgresql)
 - [Using Notion as a CMS](https://blog.railway.app/p/next-notion-blog)
-- [Getting Started (with NextJS Prisma)](getting-started)
+- [Getting Started (with NextJS Prisma)](/getting-started)
```

Comment:
This is the only relevant change, right? Can we remove all the other new lines that were added?

Reply:
hi, thanks for the feedback. Just made the changes 👍